package.json

```diff
@@ -46,7 +46,7 @@
     "semver": "~5.0.1",
     "serve-static": "~1.10.0",
     "signals": "~1.0.0",
-    "socket.io": "~0.9.16",
+    "socket.io": "~1.3.6",
```

Comment:
I believe the reason why we didn't uptick socket.io was because 0.9.16 and 1.3.6 of socket.io didn't work well with phantom.js and screw up the click tests.  It is good to know that clicktests passes with new version.

@FredrikNoren There was no other reason to prevent us from upgrading socket.io right?